dotdotbot.py

```diff
@@ -26,6 +26,6 @@ async def on_message(message):
             tempy = tempy[1:]
         tempy = message.author.nick + ' said: ' + tempy 
         requests.post("http://127.0.0.1:5000/print", json={"content": tempy})
-        await message.channel.send("printing...")
+        await message.channel.add_reaction('🖨️')
```

Comment:
I think you need to replace `message.channel.add_reaction()` with `message.add_reaction()` since the reaction is per message instead of channel-wide.
[documentation](https://discordpy.readthedocs.io/en/stable/faq.html#how-can-i-add-a-reaction-to-a-message)

Reply:
Yup, totally missed that. Fixed now